### ISSUE TITLE
How to properly use inline svg?

### ISSUE THREAD

**Issue by snaeil** [six-two/mkdocs-badges#2]
It is possible to use inline svg with your plugin as in this example:

```markdown
|<svg width="35" height="35"><circle cx="10" cy="10" r="9" stroke="green" stroke-width="2" fill="yellow" /></svg>|circle|
```

Nevertheless it not rendered in an appropriate way:

![2022-10-22-141608_121x72_scrot](https://user-images.githubusercontent.com/52743746/197338346-bd5785eb-7695-43b6-a2b5-e49d15f8a222.png)

Is it possible to appropriately use inline svg without  further hacking?
Compatibility with this would be great to be able to include even more symbols than provided through unicode.

PS: Really appreciate your plugin!

**Comment by six-two** [comment]
Hi and thanks for your comment. I think the issue here is that the SVG is to big for the badge (35px by 35px). When I make a smaller SVG (say 15px by 15px), it looks much better (see the second SVG on the [/test page](https://mkdocs-badges.vercel.app/test)).

But I will try looking into resizing the images to fit the badges or resizing the badges to fit the images.

**Comment by six-two** [comment]
The problem was, that the inline style was used. I have changed how the layout is handled, so that overlarge contents now work (see [test page](https://mkdocs-badges.six-two.dev/test/)). These fixes are published as version 0.4.0.

**Comment by snaeil** [comment]
Wow thanks for your (very quick) commitment!
